packages/core/admin/admin/src/content-manager/components/BlocksEditor/BlocksInput/index.js

```diff
@@ -74,6 +74,35 @@ const BlocksInput = ({ disabled, placeholder }) => {
     }
   };
 
+  /**
+   * Modifier keyboard shortcuts
+   */
+  const handleKeyboardShortcuts = (event) => {
+    const isMac = /Mac|iPod|iPhone|iPad/.test(navigator.userAgent);
```

Comment:
The userAgent easily be modified by users (e.g. for privacy reasons). It looks like [navigator.platform could be more solid](https://stackoverflow.com/a/11752084/3661792)

Reply:
Do you need to know? Couldn't you just look for ctrl first then cmd? Or do you both think it's too loose. My experience with user agent is it's not as reliable as you want 😓

Reply:
Navigator platform is deprecated, https://developer.mozilla.org/en-US/docs/Web/API/Navigator/platform

If we check `ctrl` first and then `cmd`, so both options will work on all the devices.